src/middlewares/register.app.js

```diff
@@ -17,6 +23,12 @@ export default (app) => {
     // Parse req object and make data available on req.body
     .use(bodyParser.json())
     .use(bodyParser.urlencoded({ extended: true }))
+    .use(session({
+      // session
+      saveUninitialized: true,
+      secret: process.env.secretOrKey
```

Comment:
Is `secretOrKey` still the name of the env variable? I think it's now `TWITTER_SESSION_KEY` but please confirm.